sdk/appconfiguration/app-configuration/README.md

```diff
@@ -175,6 +178,134 @@ async function run() {
 run().catch((err) => console.log("ERROR:", err));
 ```
 
+### Create and get a snapshot
+
+```javascript
+const appConfig = require("@azure/app-configuration");
```

Comment:
I'd destructure this: `const { AppConfigurationClient } = require("@azure/app-configuration");`